oak/common/nonce_generator.h

```diff
@@ -46,11 +46,11 @@ class NonceGenerator {
   std::random_device prng_engine_;
 };
 
-// Converts the given nonce to its base64 representation.
+// Converts the given nonce to its binary string representation.
 template <size_t N>
-std::string NonceToBase64(const Nonce<N>& nonce) {
-  absl::string_view nonce_string_view(reinterpret_cast<const char*>(nonce.data()), nonce.size());
-  return absl::Base64Escape(nonce_string_view);
+std::string NonceToBytes(const Nonce<N>& nonce) {
+  std::string nonce_bytes(reinterpret_cast<const char*>(nonce.data()), nonce.size());
+  return nonce_bytes;
```

Comment:
nit: could lose the `nonce_bytes` local variable

Reply:
Done.